docs/changelog.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 *********
 
+1.15.0 (2022-10-11)
+-------------------
+
+- Add ``go-generate`` property to the go plugin
```

Comment:
Missing an entry for the chisel slices; my suggestions:
``- Add support for Chisel slices in `stage-packages` ``